aws/resource_aws_ssm_maintenance_window.go

```diff
@@ -173,6 +185,10 @@ func resourceAwsSsmMaintenanceWindowRead(d *schema.ResourceData, meta interface{
 		return fmt.Errorf("error reading SSM Maintenance Window (%s): %s", d.Id(), err)
 	}
 
+	if err := saveTagsSSM(ssmconn, d, d.Id(), ssm.ResourceTypeForTaggingMaintenanceWindow); err != nil {
+		log.Printf("[WARN] Failed to save tags for SSM Maintenance Window (%s): %s", d.Id(), err)
```

Comment:
I think we'd prefer to return error and fail the execution here, otherwise there's very little chance of user even noticing of a potential bug (as typical user doesn't check logs) and reporting it to us.

Relatedly it may be better to move this below all other `d.Set`s as the user will end up with as much data as possible in case of a failure.